addons/udes_stock/models/stock_picking.py

```diff
@@ -77,6 +80,16 @@ class StockPicking(models.Model):
         help="Destination package (used to search on pickings)",
     )
 
+    # Highlights pickings that can be deleted for unlink()
+    u_mark = fields.Boolean(
```

Comment:
We need to review this field, it isn't very clear what it does. (We have story 953 to investigate this, but may need to decide now...)

But I'm not sure it needed for this story, the desired outcomes only mentions creating the follow on pickings, not deleting empty ones.

Will review later and get back to you.

Reply:
I guess this is the unlink of empty at the end of  push_from_drop. 
Could just comment it out and link to taiga story if not in scope.

Reply:
You won't end up with empty pickings from this without refactoring though, so we shouldn't need it all for this story I think.

Will confirm with Miquel tomorrow though.

Reply:
yep, as they are all new.